routes.jsx

```diff
@@ -15,21 +15,38 @@ import Navbar from './components/navbar/navbar.jsx';
 import Footer from './components/footer/footer.jsx';
 
 const App = React.createClass({
+  getInitialState() {
+    return {
+      suppressSplashScreen: false
+    };
+  },
   componentDidMount() {
-    setTimeout(function() {
-      this.splash.classList.add(`dismissed`);
-      document.querySelector(`#app`).classList.add(`splash-dismissed`);
-    }, 3000);
+    const suppressSplashScreenKey = `suppressSplashScreen`;
+
+    this.setState({
+      suppressSplashScreen: localStorage.getItem(suppressSplashScreenKey) === `true`
+    });
+
+    if (this.state.suppressSplashScreen !== `true`) {
+      setTimeout(function() {
+        this.splash.classList.add(`dismissed`);
+        document.querySelector(`#app`).classList.add(`splash-dismissed`);
+      }, 3000);
+      localStorage.setItem(suppressSplashScreenKey, `true`);
+    }
   },
   render() {
+
     return (
       <div>
-        <div id="splash" ref={(splash) => { this.splash = splash; }}>
-          <div className="container">
-            <h1><img src="/assets/svg/pulse-wordmark.svg" width="200" alt="Mozilla Network Pulse" /></h1>
-            <p>A stream of assets from peers across the Mozilla Network.</p>
-          </div>
-        </div>
+        { !this.state.suppressSplashScreen ?
```

Comment:
I would put this in its own function. So something like
```
{ this.renderWelcomeSplash() }
```
And then have `renderWelcomeSplash` be
```
...
renderWelcomeSplash() {
  if (this.state.suppressSplashScreen) {
    return null;
  }
  return (
    <div ...>
      ...
    </div>
  );
},
...
```